.vale/styles/RedHat/PascalCamelCase.yml

```diff
@@ -0,0 +1,51 @@
+---
+extends: existence
+scope: sentence
+ignorecase: false
+nonword: true
+level: warning
+link: https://redhat-documentation.github.io/asciidoc-markup-conventions
+message: "Consider wrapping this Pascal case or Camel case term ('%s') in backticks."
+source: https://github.com/redhat-documentation/vale-at-red-hat/tree/main/.vale/styles/RedHat/PascalCamelCase.yml
+tokens:
+  - ([A-Z]+[a-zA-Z]+){2,}
+  - ([a-zA-Z]+([A-Z][a-z]+))+
+  - Alertmanager
+  - Authentication
+  - Binding
+  - Build
+  - Config
+  - Console
+  - Deployment
+  - Endpoints
+  - Etcd
+  - Event
+  - Group
+  - Help
+  - Identity
```

Comment:
I am not sure how I feel about common words that are often used with upper case in project names. This, for example, would trigger a lot of false positives for Identity Management (IdM) in RHEL documentation (422 false positives to be precise).